scripts/fetch.sh

```diff
@@ -208,26 +225,28 @@ trap cleanup EXIT
 # any BSR cluster, so we are not immediately receiving the updated versions of the dependencies in
 # the dependent modules.
 
+# Modules from source repositories that do not have a buf.yaml configuration are defaulted to v1.
+
 # Keep this module list synced with README.md
-# sync_references ${sync_strategy} ${owner} ${repo} ${git_remote} ${opt_proto_subdir}
-sync_references releases bufbuild confluent https://github.com/bufbuild/confluent-proto
-sync_references releases bufbuild protovalidate https://github.com/bufbuild/protovalidate proto/protovalidate
-sync_references releases bufbuild protovalidate-testing https://github.com/bufbuild/protovalidate proto/protovalidate-testing
-sync_references commits bufbuild reflect https://github.com/bufbuild/reflect
-sync_references commits cncf xds https://github.com/cncf/xds
-sync_references releases envoyproxy envoy https://github.com/envoyproxy/envoy api
+# sync_references ${sync_strategy} ${owner} ${repo} ${git_remote} ${source_config_version} ${opt_proto_subdir}
```

Comment:
Is it not possible to determine the source config version using `yq` or some other means?